ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Dragonwell/tasks/main.yml

```diff
@@ -16,7 +16,7 @@
 
 - name: Download Dragonwell
   win_get_url:
-    url: 'https://github.com/alibaba/dragonwell8/releases/download/ragonwell-8.5.4_jdk8u272-ga/Alibaba_Dragonwell_8.5.4-GA_Experimental_Windows_x64.zip'
+    url: https://ci.adoptopenjdk.net/userContent/dragonwell/Windows_Dragonwell_Bootstrap.zip
```

Comment:
I am using the https://ci.adoptopenjdk.net/userContent/dragonwell/ to download, not dependent on dragonwell jenkins, this file is http://ci.dragonwell-jdk.io/userContent/dragonwell/Windows_Dragonwell_Bootstrap.zip for now, we need to put it to adopt user content

Reply:
Do you expect to need an unreleased boot JDK? Otherwise I don't see the advantage. Updating the boot JDK still requires a playbook run and preferably a checksum update in the playbook.

Reply:
The underlying question is: How often do you need to update the boot JDK? I guess it wasn't expected to be needed at all when it was requested to add the downloads to the playbooks. We rarely change the boot JDKs for all other builds.

Reply:
Yes. You're right. Update bootstrap image is a seldom case.

Reply:
@aahlenst I updated the title and issue, also found a typo in the download URL.
I have been struggling to set up remote windows with ansible, so I appreciate if I could get some help to verify this  on your windows server.

Reply:
I'm happy to merge this if it's challenging to test, since the current links are broken this can't make it any worse.

Reply:
@sxa or @Willsparker should be able to run the tests. And we should at least have a checksum for the download.

Reply:
Here ya go: https://ci.adoptopenjdk.net/job/VagrantPlaybookCheck/960/ 
+1 on the checksum

Reply:
@Willsparker We should probably look at making the HotSpot checkbox on VPC a drop down which includes Dragonwell

Reply:
> @Willsparker We should probably look at making the HotSpot checkbox on VPC a drop down which includes Dragonwell

We'll need to update `buildJDKwin.sh` and `VPC.sh` to accept them as arguments too